view/src/test/java/cmm/apps/esmorga/view/navigation/NavigationTest.kt

```diff
@@ -255,6 +258,26 @@ class NavigationTest {
         composeTestRule.onNodeWithTag(EVENT_DETAILS_EVENT_NAME).assertIsDisplayed()
     }
 
+    @Test
+    fun `given user in event details, when clicks on join event and no network available, then no network screen is shown`() {
+        val failureJoinEventUseCase = mockk<JoinEventUseCase>(relaxed = true).also { useCase ->
+            coEvery { useCase(any()) } returns EsmorgaResult.failure(EsmorgaException("No Connection", Source.REMOTE, ErrorCodes.NO_CONNECTION))
+        }
+        loadKoinModules(module { factory<JoinEventUseCase> { failureJoinEventUseCase } })
+
+        setNavigationFromDestination(Navigation.EventDetailScreen("1"))
+
+        composeTestRule.onNodeWithTag(EVENT_DETAILS_EVENT_NAME).assertIsDisplayed()
```

Comment:
There is no need to check if the screen is displayed, it's the one you are opening...